static_src/components/route_form.jsx

```diff
@@ -155,6 +158,17 @@ export default class RouteForm extends React.Component {
             className={ this.styler('route-form-preview') }
             value={ this.fullUrl }
           ></input>
+          <div style={{ paddingLeft: '.75rem' }}>
+            {(() => {
+              if (this.props.error) {
+                return <FormError message={this.props.error.description} />;
```

Comment:
just out of curiosity, what are the two sources of error for? i know where the route one comes from but what populates the `this.props.error`. 

and is it possible that there is an error in both `this.props.error` and `route.error` wondering if we should surface both of them in that case??